ci/test_e2e

```diff
@@ -25,32 +38,21 @@ if [ -n "$CF_API_ENDPOINT" ]; then
     : ${CF_ADMIN_PASSWORD?"Need to set CF_ADMIN_PASSWORD"}
```

Comment:
If the `test_e2e` is changed to only run the E2E tests, then the `CF_API_ENDPOINT` is always required. That is, you can change the conditional on line 36 from:
```
if [ -n "$CF_API_ENDPOINT" ]; then
```
to something that just verifies that it's set:
```
: ${CF_API_ENDPOINT?"Need to set CF_API_ENDPOINT"}
```